src/adera/Machines/Rocket.cpp

```diff
@@ -92,7 +101,40 @@ void SysMachineRocket::update_physics(ActiveScene& rScene)
             continue;
         }
 
-        //std::cout << "updating a rocket\n";
+        // Check for nonzero throttle, continue otherwise
+        WireData *throttle = machine.m_wiThrottle.connected_value();
+        wiretype::Percent* percent = nullptr;
+        if (throttle)
+        {
+            using wiretype::Percent;
+            percent = std::get_if<Percent>(throttle);
+            if (!(percent->m_value > 0.0f))
+            {
+                continue;
+            }
+        }
+        else { continue; }
```

Comment:
i recommend against doing one-line else statements like this, unless its matched to a 1-line if statement.

Makes the flow of the program a bit hard to anticipate.